h/static/scripts/annotator/guest.coffee

```diff
@@ -10,21 +23,21 @@ module.exports = class Guest extends Annotator
     ".annotator-adder button click":     "onAdderClick"
     ".annotator-adder button mousedown": "onAdderMousedown"
     ".annotator-adder button mouseup":   "onAdderMouseup"
+    ".annotator-hl click":               "onHighlightClick"
+    ".annotator-hl mouseover":           "onHighlightMouseover"
+    ".annotator-hl mouseout":            "onHighlightMouseout"
     "setVisibleHighlights": "setVisibleHighlights"
 
   # Plugin configuration
   options:
     TextHighlights: {}
-    EnhancedAnchoring: {}
-    DomTextMapper: {}
     TextSelection: {}
-    TextRange: {}
-    TextPosition: {}
-    TextQuote: {}
-    FuzzyTextAnchors: {}
-    FragmentSelector: {}
+
+  # Anchoring module
+  anchoring: require('./anchoring/html')
```

Comment:
I made this a first-class citizen of the the `Annotator` instance rather than something for the options hash. It feels right. I don't want the options to grow forever.